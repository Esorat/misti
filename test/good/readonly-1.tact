fun test(): Int {
    let a: Int = 10; // Warning: read-only variable
    return a;
}
