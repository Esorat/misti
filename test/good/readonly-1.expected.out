test/good/readonly-1.tact:2:5:
  1 | fun test(): Int {
> 2 |     let a: Int = 10; // Warning: read-only variable
          ^
  3 |     return a;
Read-only variable
Help: Consider creating a constant instead
See: https://nowarp.github.io/docs/misti/docs/detectors/ReadOnlyVariables